.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,16 @@
+name: Airgap
```

Comment:
Is not a name like `shellcheck` or `ci` a better one?

Reply:
done in https://github.com/kubewarden/utils/pull/3/commits/e4da1df466ce3ff1a27f4e07f881ea47363c1932